packages/paypal-commerce-integration/src/paypal-commerce/paypal-commerce-payment-strategy.spec.ts

```diff
@@ -390,6 +411,55 @@ describe('PayPalCommercePaymentStrategy', () => {
             expect(paymentIntegrationService.submitOrder).toHaveBeenCalled();
         });
 
+        it('triggers restart flow when specified error occurs', async () => {
+            const payload = {
+                payment: {
+                    methodId: defaultMethodId,
+                },
+            };
+
+            const providerError = {
+                status: 'error',
+                three_ds_result: {
+                    acs_url: null,
+                    payer_auth_request: null,
+                    merchant_data: null,
+                    callback_url: null,
+                },
+                errors: [
+                    {
+                        code: 'invalid_request_error',
+                        message:
+                            'Were experiencing difficulty processing your transaction. Please contact us or try again later.',
+                    },
+                    {
+                        code: 'transaction_rejected',
+                        message: 'Payment was declined. Please try again.',
+                        provider_error: {
+                            code: 'INSTRUMENT_DECLINED',
+                        },
+                    },
+                ],
+            };
+
+            jest.spyOn(paypalCommerceIntegrationService, 'submitPayment').mockImplementation(() => {
+                // eslint-disable-next-line @typescript-eslint/no-throw-literal
```

Comment:
Would be better to add this rule for `*.spec.ts` in `.eslintrc.json`

Reply:
I see you added this rule for these formats `["*.ts", "*.tsx", "*.js", "*.jsx"]` but since you only use it for testing purposes, i mean only for `*.spec.ts` you need to do it a little differently

Please look at the example in the link above.

Reply:
moved to spec section